src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -1882,499 +1801,498 @@
+  "botania.page.jadedAmaranthus0": "You likes flowers, right? If so, $(1)Jaded Amaranthus$(0) is made just for you.$(p)Feeding it with $(4)Mana$(0) from a $(1)Mana Pool$(0) will have it grow new $(1)Mystical Flowers$(0) around it at a rapid pace, on any available soil.",
   "botania.page.jadedAmaranthus1": "$(o)It was this or clematis.$()",
-  "botania.page.jadedAmaranthus2": "$(o)Wait a minute, did you just summon a bunch of flowers in one turn?$(p)Yeah, so?$(p)That's against the rules, isn't it?$(p)Screw the rules, I have Mana!",
+  "botania.page.jadedAmaranthus2": "$(o)Wait a minute, did you just summon a bunch of flowers in one turn?$(p)Yeah, so?$(p)That's against the rules, isn't it?$(p)Screw the rules, I have mana!",
 
   "botania.entry.bellethorne": "Bellethorne",
   "botania.tagline.bellethorne": "Damages mobs",
-  "botania.page.bellethorne0": "The $(1)Bellethorne$(0) is a malevelous flower. By utilizing $(4)Mana$(0) it slowly inflicts harm in any living beings (save for $(4)players$(0)) around it.",
+  "botania.page.bellethorne0": "The $(1)Bellethorne$(0) is a malevelous flower. It twists any $(4)Mana$(0) provided to slowly inflict harm in any living beings (save for $(4)players$(0)) around it.",
   "botania.page.bellethorne1": "$(o)Every rose has its thorn$().",
 
   "botania.entry.dreadthorne": "Dreadthorne",
   "botania.tagline.dreadthorne": "Damages adult animals",
-  "botania.page.dreadthorne0": "The $(1)Dreadthorne$(0) is a slightly more specific counterpart to the $(1)Bellethrorne$(0), by which, instead of all mobs around, it'll only hurt $(4)adult$(0) animals and none else.",
+  "botania.page.dreadthorne0": "The $(1)Dreadthorne$(0) is a slightly more specialized counterpart to the $(1)Bellethorne$(0): more specifically, instead of $(o)all$() nearby beings, it only harms $(4)adult animals$(0).",
   "botania.page.dreadthorne1": "$(o)It's a modus operandi$().",
 
   "botania.entry.heiseiDream": "Heisei Dream",
   "botania.tagline.heiseiDream": "Turns mobs against each other",
-  "botania.page.heiseiDream0": "The $(1)Heisei Dream$(0) will, by utilizing $(4)Mana$(0), awaken feelings of anger and revenge in any nearby $(4)mobs$(0), turning them against each other.",
+  "botania.page.heiseiDream0": "The $(1)Heisei Dream$(0) uses $(4)Mana$(0) to fuel artificial emotions of anger and revenge in any nearby $(4)mobs$(0), turning them against each other.",
   "botania.page.heiseiDream1": "$(o)A pleasant era will murder people$().",
 
   "botania.entry.tigerseye": "Tigerseye",
   "botania.tagline.tigerseye": "Scares and repels creepers",
-  "botania.page.tigerseye0": "It's a known fact that $(4)Creepers$(0) are terrified of $(4)cats$(0).$(p)The $(1)Tigerseye$(0) exploits that very weakness, by emitting a feline aura, petrifying any nearby $(4)Creepers$(0), preventing them from exploding. It also has the effect of making t
```

Comment:
Manually placed implies some other effect when automated. (does this comment show you on Tigerseye instead of Dandelifeon?)